app/src/main/java/rick_and_morty/eventbus/EventBus.kt

```diff
@@ -0,0 +1,18 @@
+package rick_and_morty.eventbus
+
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableSharedFlow
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class EventBus @Inject constructor() {
+
+    private val bus = MutableSharedFlow<BusEvent>(extraBufferCapacity = 64)
+
+    val events: Flow<BusEvent> = bus
```

Comment:
HINT: Usually the backing property of a immutable property has the same name starting with an underscore.
Meaning `bus` should become `_events`.
Additional info: https://stackoverflow.com/questions/6903022/why-do-we-use-in-variable-names 

Also, you probably want to increase the `extraBufferCapacity` to `Int.MAX_VALUE` to ensure that there will almost never be a bottleneck situation where events get lost.

Reply:
Done